### ISSUE TITLE
CustomImportOrder accepts undocumented "tokens" property

### ISSUE THREAD

**Issue by stephengold** [checkstyle/checkstyle#21443]
I have read check documentation: https://checkstyle.org/checks/imports/customimportorder.html
I have downloaded the latest Checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the CLI and shown it below, as CLI describes the problem better than 1,000 words.

```bash
$ javac MyClass.java
$ cat my_checks.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
  <module name="TreeWalker">
    <module name="CustomImportOrder">
      <property name="tokens" value="IMPORT, STATIC_IMPORT, PACKAGE_DEF"/>
    </module>
  </module>
</module>
$ RUN_LOCALE="-Duser.language=en -Duser.country=US"
$ java $RUN_LOCALE -jar checkstyle-14.1.0-all.jar -c my_checks.xml MyClass.java
Starting audit...
Audit done.
$ 
```
---

I expect the "tokens" property to be documented, or else the audit should fail with a `CheckstyleException`.

**Comment by romani** [comment]
https://github.com/checkstyle/checkstyle/blob/0bfa5972ff36ef3501f6684814dd43c4abb0e062/src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java#L337-L342

This is the the only tokens that are possible. User can re set them all or set no of them. It works the same.

If you put any other token, it will fail as you expects.

We do not document tokens at https://checkstyle.org/checks/imports/customimportorder.html#Properties as users has no way to change them other than repeating what is the only possible set.

https://checkstyle.org/writing-checks.html#Understanding_token_sets

**Comment by stephengold** [comment]
> We do not document tokens at https://checkstyle.org/checks/imports/customimportorder.html#Properties as users has no way to change them other than repeating what is the only possible set.

Thank you @romani for helping understand how CustomImportOrder works. It's helpful to know that only one set of tokens is possible for this module. The website would be improved if important facts like this one were included in the check documentation, rather than being buried in the source code.

Is there a separate channel for raising documentation issues?

**Comment by romani** [comment]
Let's create separate issue for doc update.
